app/src/main/java/org/dhis2/data/forms/dataentry/ValueStoreImpl.kt

```diff
@@ -131,8 +131,8 @@ class ValueStoreImpl(
                 EntryMode.DE -> {
                     val event = d2.eventModule().events().uid(recordUid).blockingGet()
                     val enrollment = d2.enrollmentModule().enrollments()
-                        .uid(event.enrollment()).blockingGet()
-                    enrollment.trackedEntityInstance()
+                        .uid(event?.enrollment()).blockingGet()
```

Comment:
shouldn't launch an exception uid(NULL)?. maybe is better `event?.let { d2.enrollmentModule().enrollments().uid(it) }`